<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMuPDFReaderClass</class>
 <widget class="QWidget" name="QMuPDFReaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMuPDFReader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">#frame
{
	background: #1E1E20;
	/*border-radius: 6px;*/
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_top" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel
{
	font-family: 微软雅黑;
	font-size: 14px;
	color: #FFFFFF;
}

QPushButton
{
	border-radius: 14px;
	border: 1px solid #01EEC3;
	font-size: 12px;
	font-family: 微软雅黑;
	color: #01EEC3;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_pdfFileName">
           <property name="styleSheet">
            <string notr="true">	font-family: 微软雅黑;
	font-size: 14px;
	color: #FFFFFF;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_openPDF">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/image/qrc/image/Folder_dev.png</normaloff>:/image/qrc/image/Folder_dev.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_prevPage">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_nextPage">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pageInfo">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgba(234, 234, 234, 0.15);
border-radius: 15px;
font-size: 12px;
font-family: 微软雅黑;
color: #FFFFFF;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>输入页码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_goToPage">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>跳转</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_zoomIn">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>放大</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_zoomFactor">
           <property name="text">
            <string>100%</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_zoomOut">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>缩小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_printer">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>打印</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_close">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>28</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_pdfPage">
           <property name="styleSheet">
            <string notr="true">font-size: 12px;
font-family: 微软雅黑;
color: #FFFFFF;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="styleSheet">
         <string notr="true">QScrollArea { border:none; background: transparent; }
QScrollArea &gt; QWidget &gt; QWidget { background: transparent; }
QScrollArea &gt; QWidget &gt; QScrollBar { background: palette(base); }</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="SequentialPageWidget" name="pdfPages">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1019</width>
           <height>728</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SequentialPageWidget</class>
   <extends>QWidget</extends>
   <header location="global">sequentialpagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QMuPDFReader.qrc"/>
 </resources>
 <connections/>
</ui>
